parm/aqm.rc

```diff
@@ -29,11 +29,9 @@ ctm_pmdiag: true
 #
 #  Input emissions
 #
-{%- if do_aqm_canopy %}
-emission_sources: anthro fire canopy
-{%- else %}
 emission_sources: anthro fire
-{%- endif %}
+{%- if do_aqm_canopy %} canopy{% endif %}
+{%- if do_aqm_fengsha %} fengsha{% endif %}
 
```

Comment:
@zmoon, with this `fengsha` flag, the `run_fcst' task fails. I think the line 34 should be removed. If you succeeded with this flag, there should be any other changes somewhere else. Please check this again.

Reply:
@chan-hoo I believe that this is a different issue as 1) without this fengsha is not actually being run as `fengsha` is not included as an emission source.  2) the file being read in within the sample config has the wrong dimensions and will crash.  I'm running a test right now with the updated file that should fix this issue. I'll create a separate issue for this once I have a day completed to show.

Reply:
@bbakernoaa, if so, you can put your change into this PR. Or, remove Ln 34 from this PR and put it into your new PR. In any cases, this `fengsha` flag should come with the valid configuration.

Reply:
@chan-hoo I completely agree I'll link the issue (which is just the input file) to this PR and get it put into the sample configuration files

Reply:
On a related note, the EMC sample configs have `DO_AQM_FENGSHA: true` but we have false. Do we want things to stay that way?

Reply:
@zmoon, good point! I am not sure. Based on your comments, I think the `fengsha` option does not work correctly in the current status of the workflow. Is this correct? @JianpingHuang-NOAA, are you aware of this issue?

Reply:
@chan-hoo Barry said that with the fixed input file it works, we will get that fix in here soon.

Reply:
@chan-hoo I went ahead and updated the path in the sample configs based on Barry's note in https://github.com/NOAA-EMC/AQM/issues/36

Reply:
@zmoon, got it. Thanks. I'll test it soon.

Reply:
it looks working well.

Reply:
> On a related note, the EMC sample configs have `DO_AQM_FENGSHA: true` but we have false. Do we want things to stay that way?

@bbakernoaa do you want to change this in our sample configs moving forward? or leave it this way?

Reply:
We should definately add this to the arl sample config.  We are the the developers of this.....

Reply:
Ok, I can add that to our branch when I update it today. Do we want to also do a PR for that to here?